broker-core/src/main/java/org/apache/qpid/server/queue/AbstractQueue.java

```diff
@@ -3520,23 +3521,46 @@ public MessageEnqueueRecord getEnqueueRecord()
                                                                           destination,
                                                                           parseSelector(selector),
                                                                           limit);
-        _virtualHost.executeTransaction(transaction);
+        executeTransactionAndLogResults("copyMessages", destination.getName(), transaction);
         return transaction.getModifiedMessageIds();
 
     }
 
+    private void executeTransactionAndLogResults(final String msg,
```

Comment:
The parameter name 'msg' needs to be changed to operationName
Let move transaction parameter to a second position
the destinationName can be renamed into a targetdestinationName

Reply:
this method no longer exists!